canonical-web-and-design/ubuntu.com
# 5135 fix get in touch formid and lpid for contact forms

## Done
- Load #get-in-touch form on button click
- Create new #get-in-touch for openstack on pricing/consulting
## QA
- Check out this feature branch
- Run the site using the command ./run serve
- View the site locally in your web browser at: pricing/consulting
- Run through the following QA steps
- Go to pricing/consulting and click the "Get in touch" button in the OpenStack section. Compare with copy doc. If you press any other "Get in touch" button on this page, the form should be different.
- Go to any page that has a "Get in touch" and make sure the form is displayed on button click.
- Make sure the formid, lpid, returnurl & lpurl are the same as the once we currently use
## Issue / Card
Fixes #5315
